girder_worker/app.py

```diff
@@ -246,4 +255,6 @@ class _CeleryConfig:
     backend=backend_uri, broker=broker_uri,
     task_cls='girder_worker.app:Task')
 
+app.argument = argument
```

Comment:
Let's keep the argument decorator (and the rest of the 'describe' api) completely separate from the celery object and just directly assign _girder_description to the function objects.  that way this can be generically used across different applications (e.g. celery,  docker,  cli,  etc).